test/src/cli/commands/install-browsers/index.ts

```diff
@@ -0,0 +1,161 @@
+import path from "path";
+import { Command } from "@gemini-testing/commander";
+import proxyquire from "proxyquire";
+import sinon, { type SinonStub } from "sinon";
+import { Testplane } from "../../../../../src/testplane";
+import type { Writable } from "type-fest";
+import type { Config } from "../../../../../src/config";
+
+describe("cli/commands/install-browsers", () => {
+    const sandbox = sinon.createSandbox();
+
+    let cli: { run: () => void };
+    let loggerStub: { log: SinonStub; warn: SinonStub; error: SinonStub };
+    let testplaneStub: Writable<Testplane>;
+    let installBrowsersWithDriversStub: SinonStub;
+
+    const installBrowsers_ = async (argv: string = ""): Promise<void> => {
+        process.argv = ["foo/bar/node", "foo/bar/script", "install-browsers", ...argv.split(" ")].filter(Boolean);
+        cli.run();
+
+        await (Command.prototype.action as SinonStub).lastCall.returnValue;
+    };
+
+    const mkBrowser_ = (browserName: string, browserVersion: string): Config["browsers"][string] =>
+        ({
+            desiredCapabilities: { browserName, browserVersion },
+        } as Config["browsers"][string]);
+
+    beforeEach(() => {
+        loggerStub = { log: sandbox.stub(), warn: sandbox.stub(), error: sandbox.stub() };
+        testplaneStub = Object.create(Testplane.prototype);
+
+        Object.defineProperty(testplaneStub, "config", {
+            value: { browsers: {} },
+            writable: true,
+            configurable: true,
+        });
+
+        sandbox.stub(Testplane, "create").returns(testplaneStub as Testplane);
+
+        sandbox.stub(process, "exit");
+
+        sandbox.spy(Command.prototype, "action");
+
+        installBrowsersWithDriversStub = sandbox.stub();
+
+        cli = proxyquire("../../../../../src/cli", {
+            [path.resolve(process.cwd(), "src/cli/commands/install-browsers")]: proxyquire(
```

Comment:
no, you can't do that. Double `proxyquire` nesting is too much :)

Reply:
Just copied it from here: https://github.com/gemini-testing/testplane/blob/master/test/src/cli/commands/list-tests/index.ts#L39-L50

Is there any better way?

Reply:
yes - integration test ;)